google-cloud-storage/lib/google/cloud/storage/file/signer_v2.rb

```diff
@@ -24,7 +24,7 @@ module Storage
       class File
         ##
         # @private Create a signed_url for a file.
-        class Signer
+        class SignerV2
```

Comment:
I'd add a require Addressable in this file too.

Reply:
I think this should happen in #3081. I'll request it there.